LunaDll/Input/LunaGameController.h

```diff
@@ -115,6 +117,8 @@ class LunaGameController
     unsigned int axisPadState;
     unsigned int padState;
     unsigned int buttonState;
+    unsigned int xAxis;
+    unsigned int yAxis;
```

Comment:
Also, I don't see the point in making `xAxis` and `yAxis` unsigned ints since they receive `int` values in `LunaGameController::controllerAxisEvent` and `getStickX` and `getStickY` casts them to `int`.